fabric-events-lifecycle-v0/build.gradle

```diff
@@ -1,5 +1,5 @@
 archivesBaseName = "fabric-resource-loader-v0"
-version = getSubprojectVersion(project, "0.1.1")
+version = getSubprojectVersion(project, "0.1.2")
```

Comment:
why did you bump the resource loader version?

Reply:
uh oh

Reply:
lemme do a real quick fix

Reply:
that's the wrong archivesbasename lemme fix